contract-tests/README.md

```diff
@@ -0,0 +1,5 @@
+# SSE client contract test service
+
+This directory contains an implementation of the cross-platform SSE testing protocol defined by https://github.com/launchdarkly/sse-contract-testing. See that project's `README` for details of this protocol, and the kinds of SSE client capabilities that are relevant to the contract tests. This code should not need to be updated unless the SSE client has added or removed such capabilities.
```

Comment:
This is a draft, so commenting may be pre-mature, but I don't think this URL is correct.
 I see: https://github.com/launchdarkly/sse-contract-tests but not the URL specified in this comment (testing ->tests).

Reply:
Ugh, indeed

Reply:
Of course I pasted that same wrong content into a half-dozen repos.